### DISCUSSION TITLE
Add the ability to hide drag space for vertical tab optimized layout

### DISCUSSION THREAD

**Discussion by lnkd61** [Floorp-Projects/Floorp#960] [Feature Request]
The recent update for Floorp (11.10.5) has introduce an area above the navigation bar when using the "Optimise browser for Vertical Tab Bar" option. I believe this is an intentional change, however I would like the option to disable this space as the layout that I am using already has drag space. 

![image](https://github.com/Floorp-Projects/Floorp/assets/1360980/f59d2614-aec4-4549-ace7-0ed376ae7261)

**Comment by CutterKnife** [comment]
Will be added in the future.
